### ISSUE TITLE
Recipe generator

### ISSUE THREAD

**Issue by PierrePaul** [Chi-teck/drupal-code-generator#182]
Hello everyone!

I wanted to know if there was some interest into a recipe generator in the drupal-code-generator.
So far I've found https://www.drupal.org/project/recipe_generator which has not been updated in a while and no longer works with the new generator system (and it doesn't have any unit test).

I've done the first version of the SDC generator, so I should be able to tackle this, but I want to make sure there's interest to have it directly in drupal-code-generator project, or if it should be kept as a separate module for now.

Thanks!

**Comment by Chi-teck** [comment]
The main concern here is maintenance.  It's hard to maintain something that you are not using yourself. Consider SDC command, there is a couple follow-ups that needs to be addressed.

https://github.com/Chi-teck/drupal-code-generator/pull/181
https://github.com/Chi-teck/drupal-code-generator/issues/115

**Comment by PierrePaul** [comment]
I'm not exactly sure how to respond to that. 

If you want to assign any new SDC related issues to me, that would be really cool.

I feel like 2 issues in ~ 1 year and half isn't so bad. Especially if you consider one as invalid.

**Comment by Chi-teck** [comment]
#115 was requested again in Drush (https://github.com/drush-ops/drush/issues/6132). Seems like it might be useful.

**Comment by Chi-teck** [comment]
As of recipe generator I am not sure yet. We actually thinking about dropping some generators to reduce maintenance cost (see #155).

**Comment by Chi-teck** [comment]
Checked https://github.com/kevinquillen/recipe-generator/tree/1.0.x. Seems like it's not too complex.

**Comment by PierrePaul** [comment]
Yeah I think #115 would be useful. I know quite a few agencies that would like to have it. They usually try to have a base module that they can re-use across their sites, including some of them that already have a theme imposed. I'm up to work on it if there's a consensus about it.

And no, the recipe generator is not too complex at this point. Migrating to the new version isn't going to be too hard, but if we want to reduce maintenance... Recipes is still a work in progress as well, so the spec is likely to change too.
